templates/main/modules/browse.html

```diff
@@ -26,6 +27,7 @@ <h3>No tests available. Start by <a href="import.php">importing</a> one!</h3>
             </tr>
             <?php foreach ($params["tests"] as $index => $test): ?>
                 <tr>
+                	<td><a href="test.php?id=<?=$test->id?>">Export</a></td>
```

Comment:
Hm, what do you think about putting it to the right side of the row? Also, maybe use an `img` here?